pyat/at/physics/rdt.py

```diff
@@ -0,0 +1,545 @@
+from __future__ import annotations
+
+import numpy
+import multiprocessing
+from functools import partial
+from ..lattice import Lattice, Refpts, Multipole, All
+from enum import Enum
+from collections.abc import Sequence
+from dataclasses import dataclass, asdict
+
+
+__all__ = ["get_rdts", "RDTType"]
+
+
+class RDTType(Enum):
+    """Enum class for RDT type"""
+
+    ALL = 0
+    FOCUSING = 1
+    COUPLING = 2
+    CHROMATIC = 3
+    GEOMETRIC1 = 4
+    GEOMETRIC2 = 5
```

Comment:
GEOMETRIC1 and GEOMETRIC2 include different orders. But it is not described.
Please add the details on what is calculated on each.